src/main/java/command/DeleteCommand.java

```diff
@@ -0,0 +1,46 @@
+package command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import task.Task;
+
+/**
+ * Command to delete a Task.
+ *
+ * @author felix-ong
+ */
+public class DeleteCommand extends Command {
+    private String index;
+    private Task task;
+
+    public DeleteCommand(String index) {
+        this.index = index;
+    }
+
+    /**
+     * Executes the specific actions for this command.
+     *
+     * @param tasks   Handles the list of tasks.
+     * @param storage Handles the saving and loading of tasks.
+     */
```

Comment:
Perhaps you could stick to the same style for the javadoc comments of the param?
```suggestion
     * Executes the specific actions for this command.
     *
     * @param tasks Handles the list of tasks.
     * @param storage Handles the saving and loading of tasks.
     */
```